tests/FSharp.Compiler.ComponentTests/EmittedIL/StringFormatAndInterpolation.fs

```diff
@@ -88,4 +88,46 @@ IL_0014:  call       string [runtime]System.String::Concat(string,
                                                                   string,
                                                                   string,
                                                                   string)
-IL_0019:  ret"""]
\ No newline at end of file
+IL_0019:  ret"""]
+
+    [<FSharp.Test.FactForNETCOREAPP>]
+    let ``Interpolated string with concat converts to span implicitly`` () =
+        let compilation = 
+                FSharp $"""
+        module InterpolatedStringByefLikes
+        type Foo() =
+
+                let sb = System.Text.StringBuilder()
+
+                member _.Bar(s: System.ReadOnlySpan<char>) = sb.Append(s) |> ignore
+
+        let [<EntryPoint>] main _ =
+            let foo = Foo()
+            let foos = "foo"
+            foo.Bar($"{{foos}} is bar")
+            0
+                """
+
+        compilation |> compile |> shouldSucceed |> ignore
+
+        compilation |> asExe |> compileAndRun |> shouldSucceed |> ignore
```

Comment:
Why is this getting compiled three times?
Can be done just once, and then:
- run
- verify
The same compilation result

Reply:
I just decided to do independent compilations. It probably can. I don't want to change it now though, since I don't want to wait for another run and postpone release build for another hour or so.